src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -38,17 +38,24 @@ export interface IFacetSliderOptions extends ISliderOptions {
 }
 
 /**
- * The FacetSlider component allows to create a facet that renders a slider widget to filter on a range of numerical values
- * rather than the classic multi-select facet with a label and a count for each values.<br/>
- * Note that this component does *NOT* inherit from a standard {@link Facet}, and thus does not offer all the same options.
- * Also note that many options for the slider component cannot be set as an HTML attribute on the component, and must be configured in JavaScript.
+ * The FacetSlider component creates a facet containing a slider widget that allows the end user to filter results based
+ * on a range of numerical values, rather than a "classic" multi-select {@link Facet} with a label and a count for each
+ * value.
  *
- * ## A generic example on how to initialize a complex slider
- * ```
- * // Using a JSON object inside the init call. Note that the JSON object follows the options described in this page.
+ * Note that this component does not inherit from the Facet component, and thus does not offer the same configuration
+ * options. Also, some of the FacetSlider options cannot be set as a HTML attributes on the component and must rather be
+ * configured in the {@link init} call of the search interface.
+ *
+ * **Examples:**
+ *
+ * Specifying the FacetSlider configuration using a JSON inside the init call. Note that the JSON follows the
+ * FacetSlider options:
+ *
+ * ```javascript
+ * // You can call the init script using "pure" JavaScript:
  * Coveo.init(document.querySelector('#search'), {
  *    FacetSlider: {
- *      field: "@size",
+ *      field: "@syssize",
```

Comment:
Never use @sys fields in documentation. @sys fields are deprecated (still works, but deprecated)